ui/theme.ts

```diff
@@ -111,7 +112,7 @@ const globalStyles = css({
     fontSize: 16, // baseline for rems
     minHeight: '100vh',
     scrollBehavior: 'smooth',
-    bg: 'bluegrey.50',
+    bg: isRcvrEnv ? 'bluegrey.50' : backgroundColor,
```

Comment:
Why do you use a ternary here instead of putting it into `whitelabels.ts` like for the other envs?

Reply:
Good question. Will change that.